### ISSUE TITLE
Fails if there is a variable named module

### ISSUE THREAD

**Issue by smirea** [gaearon/react-hot-loader#812]
If you are reporting a bug or having an issue setting up React Hot Loader, please fill in below. For feature requests, feel free to remove this template entirely.

### Description

A runtime error occurs if there is a variable named `module` in the file

### Expected behavior

It should work regardless of any variable names

### Actual behavior

A runtime error is thrown

### Environment

React Hot Loader version: 4.0.0-beta.16

Run these commands in the project folder and fill in their results:

1. `node -v`: 7.10.1
2. `npm -v`: 4.2.0

Then, specify:

1. Operating system: OSX
2. Browser and version: Chrome latest

### Reproducible Demo

example file:

```js
import React from 'react';

export const WorkingComponent = () => <div>hi!</div>

export const module = () => console.log('this should work');
```

Adding the above file to the pipeline will fail to compile

**Comment by theKashey** [comment]
Ok. Got the point. You are shadowing the build-in module variable, and we are not expecting it.
And the `real` variable does not exist due to hoisting.

The only way - just dont fall in case of error.

**Comment by neoziro** [comment]
Fixed in [v4.0.0-beta.17](https://github.com/gaearon/react-hot-loader/releases/tag/v4.0.0-beta.17).
